app/assets/stylesheets/application/panel.scss

```diff
@@ -0,0 +1,16 @@
+.panel {
+  .actions {
+    form, button {
+      display: inline-block;
+    }
+  }
+
+  #rating, .author {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line